portality/settings.py

```diff
@@ -681,7 +687,7 @@
             "auth" : False,
             "role" : None,
             "query_validator" : "public_query_validator",
-            "query_filters" : ["only_in_doaj", "last_update_fallback"],
+            "query_filters" : ["only_in_doaj", "last_update_fallback", "search_all_meta"],
```

Comment:
We also need to apply this constraint to the `journal,article` query route, otherwise that will allow you to search through notes via that endpoint (which is public).  Not sure on the implications of that, as articles do not have an `all_meta` field

Reply:
Hm, that could be a problem, because this would allow the notes data to "leak" through the fixed-query-widget's endpoint, so we need to think of a solution.  I don't really want to add an `all_meta` to articles unnecessarily, but it might be the only way.  Thoughts?